.github/actions/delete-control-plane-app/delete-app.sh

```
@@ -0,0 +1,38 @@
+
+# Script to delete a Control Plane application
+# Required environment variables:
+# - APP_NAME: Name of the application to delete
+# - CPLN_ORG: Organization name
+
+set -e
+
+printenv
+
+# Validate required environment variables
+: "${$APP_NAME:?APP_NAME environment variable is required}"
```

Shellcheck (0.10.0)

[error] ${$x} is invalid. For expansion, use ${x}. For indirection, use arrays, ${!x} or (for sh) eval.

(SC2298)